website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -35,12 +35,16 @@ When using defer, it compares artifacts from the most recent successful producti
 
 ### Defer in the dbt Cloud IDE
 
+For deferral in the IDE, you must have production artifacts available generated by a deploy job. dbt Cloud will first check for these artifacts in your Staging environment (if it exists) or, if not, the Production environment. 
+
+The defer feature in the IDE is not expected to work if a Staging environment exists but no deploy job has run there. The necessary metadata to power the defer feature will not exist until a deploy job has run successfully in the Staging environment.
```

Comment:
```suggestion
The defer feature in the IDE wont't work if a Staging environment exists but no deploy job has run. This is because the necessary metadata to power the defer feature won't exist until a deploy job has run successfully in the Staging environment.
```